debug/gdbserver.py

```diff
@@ -710,6 +715,7 @@ def test(self):
             self.gdb.p("$pc")
             assertRegex(output, r"[bB]reakpoint")
             assertIn("rot13 ", output)
+            self.gdb.stepi()
```

Comment:
why do we need this `stepi` ?

Reply:
https://github.com/riscv-software-src/riscv-tests/blob/7dfa49bb33ac2393742232cb6f77b79ec5c9df47/debug/programs/debug.c#L65-L68
According to the program, I think that the breakpoint should be hit 2 times at rot13, and then hit at crc32a and _exit sequentially. So stepi avoids hitting the rot13 breakpoint at the same position as main

Reply:
> So stepi avoids hitting the rot13 breakpoint at the same position as main

I don't get it. What do you mean "at the same position as main" ?  main and rot13 should have different addresses. When gdb issues a continue request it issues a single-step and then issues a real continue. We should breakpoint at main should not be hit.